### DISCUSSION TITLE
建议从 看图 复制的图像可以粘贴到gimp里

### DISCUSSION THREAD

**Discussion by fishBone000** [linuxdeepin/developer-center#5068] [Features Request & Ideas | 特性请求 & 头脑风暴]
目前在看图里右键图片点复制，再到GIMP里粘贴会提示剪贴板中无图像内容，虽然说粘贴到QQ就没问题。
建议让GIMP也能粘贴到图片，能省事很多

Deepin 20.9 社区版
看图 5.9.10

**Comment by BLumia** [comment]
虽说我觉得这个算 GIMP 的支持问题，不过看图如果像旧版本一样，把复制文件和仅复制图片内容的选项拆开的话，估计就会好一些，避免应用程序猜错用户想粘贴的东西到底是什么。

cc @meiyixiang
